aviary/validation_cases/benchmark_tests/test_bench_GwGm.py

```diff
@@ -13,26 +12,26 @@
 @use_tempdirs
 class ProblemPhaseTestCase(unittest.TestCase):
 
-    @require_pyoptsparse(optimizer="SNOPT")
+    @require_pyoptsparse(optimizer="IPOPT")
     def bench_test_swap_2_GwGm(self):
         local_phase_info = deepcopy(phase_info)
         prob = run_aviary('models/test_aircraft/aircraft_for_bench_GwGm.csv',
-                          local_phase_info)
+                          local_phase_info, optimizer='IPOPT')
 
         rtol = 0.01
 
         # There are no truth values for these.
         assert_near_equal(prob.get_val(Mission.Design.GROSS_MASS),
-                          174039., tolerance=rtol)
+                          181654., tolerance=rtol)
 
         assert_near_equal(prob.get_val(Aircraft.Design.OPERATING_MASS),
-                          95509, tolerance=rtol)
+                          101555., tolerance=rtol)
 
         assert_near_equal(prob.get_val(Mission.Summary.TOTAL_FUEL_MASS),
-                          42529., tolerance=rtol)
+                          44098., tolerance=rtol)
 
         assert_near_equal(prob.get_val('landing.' + Mission.Landing.GROUND_DISTANCE),
-                          2634.8, tolerance=rtol)
```

Comment:
where did these updated values come from?

Reply:
running the benchmark on main